ee/dtr/user/manage-images/sign-images/trust-with-remote-ucp.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Using Docker Content Trust with a Remote UCP Cluster
+description: Learn how to use a single DTR's trust data with remote UCPs.
+keywords: registry, sign, trust, notary
+---
+
+For more advanced deployments, you may want to share one Docker Trusted Registry
+across multiple Universal Control Planes. However, customers wanting to adopt
+this model, alongside the [Only Run Signed
+Images](../.../../ucp/admin/configure/run-only-the-images-you-trust.md) feature
+of UCP, run into problems as each Universal Control Plane operates an
+independent set of users.
+
+Docker Content Trust with a Remote UCP gets around this problem, as User's from
+a remote UCP are able to sign images in the central DTR, and still apply runtime
+enforcement.
+
+In the following example we will connect DTR to 1 remote UCP cluster, however
```

Comment:
@ollypom - Can we call out that if a user wants the signed image to be used across multiple clusters and if those clusters have Docker Content Trust enabled in UCP that the client bundle used to sign the image must be present in each cluster.

Also, if the DCT rules in UCP state that the signer must be in a specific org/team that the client bundle used is allocated correctly.